src/EditorFeatures/Core/Implementation/RenameTracking/AbstractRenameTrackingCodeFixProvider.cs

```diff
@@ -29,7 +29,9 @@ internal abstract class AbstractRenameTrackingCodeFixProvider : CodeFixProvider
 
         public sealed override ImmutableArray<string> FixableDiagnosticIds
         {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
what is this warning about? I don't see any obsolete?

Reply:
Thanks, I forgot to remove these lines when I removed `[Obsolete]` from `RenameTrackingDiagnosticAnalyzer`.